README.md

```diff
@@ -9,12 +9,23 @@
 	<br>
 </h1>
 
-> Repo for Filecoin Station Core
+> Station Core is a headless version of Filecoin Station suitable for running on
+> all kinds of servers.
 
 [![CI](https://github.com/filecoin-station/core/actions/workflows/ci.yml/badge.svg)](https://github.com/filecoin-station/core/actions/workflows/ci.yml)
 
+## Deployment
+
+Station Core supports different deployment options:
+
+- [Docker](#docker)
+- [Manual Deployment (Ubuntu)](#manual-deployment-ubuntu)
+
 ## Installation
 
+Station Core requires Node.js version 18.12 or newer. We recommend using the
```

Comment:
see Node.js version discussion

Reply:
Resolved in https://github.com/filecoin-station/core/pull/47/commits/5cb8a745a1c020e995bfffc2b7c12acc118f806b